daisy/step_create_instances.go

```diff
@@ -55,11 +56,18 @@ Loop:
 					}
 				}
 				if numErr > 10 {
-					w.LogStepInfo(s.name, "CreateInstances", "Instance %q: error getting serial port: %v", i.Name, err)
+					// Quit silently if we were able to read *some* data from the instance,
+					// since there's a race condition where an instance can shut down
+					// fast enough that the call to InstanceStatus will return a 404.
```

Comment:
Move this below the if. Otherwise, it appears as if this comment applies to the if statement, when in fact it applies to the nop outside.

Reply:
So, the comment is intended to explain the if block, and the circumstances under which we expect it to be executed ("*if* we were able"). I changed the wording of th comment, however, to focus on logging rather than quitting.